elastic/kibana
# [SR] Fix bug when editing retention of slm policy

Possibly related to https://github.com/elastic/kibana/issues/66798

### How to test
To reproduce the bug on master:

1. Create a SLM policy. For step 3, define a minimum or maximum count under the "Snapshots to retain" section.
2. Edit the policy created. Navigate to step 3 and add a value to the "Delete after" field. Do not change the unit dropdown (days should be selected by default).
3. Navigate to the review step, review the request. Note that the `expire_after` field is not set even though one was defined.

With this change, the `expire_after` field should be set if defined and default to the `d` time unit.

### Release note
This fixes a bug in Snapshot and Restore where the retention expiration was not always set.
